modin/numpy/arr.py

```diff
@@ -1469,6 +1559,253 @@ def sum(
                 + initial
             )
 
+    def all(self, axis=None, out=None, keepdims=None, *, where=True):
+        check_kwargs(keepdims=keepdims, where=where)
+        target = where.where(self, True) if isinstance(where, array) else self
+        if self._ndim == 1:
+            if axis == 1:
+                raise numpy.AxisError(1, 1)
+            result = target._query_compiler.all(axis=0)
+            if keepdims:
+                if out is not None and out.shape != (1,):
+                    raise ValueError(
+                        f"operand was set up as a reduction along axis 0, but the length of the axis is {out.shape[0]} (it has to be 1)"
+                    )
+                return fix_dtypes_and_determine_return(
+                    result, 1, bool, out, where is not False
+                )
+            result = result.to_numpy()[0, 0]
+            return result if not where else result and where
+        if axis is None:
+            result = target._query_compiler.all(axis=1).all(axis=0)
+            if keepdims:
+                if out is not None and out.shape != (1, 1):
+                    raise ValueError(
+                        f"operand was set up as a reduction along axis 0, but the length of the axis is {out.shape[0]} (it has to be 1)"
+                    )
+                return fix_dtypes_and_determine_return(
```

Comment:
Same here with returning the correct value when keep dims is True, but out is None.